app/screens/DishPage.js

```diff
@@ -1,342 +1,435 @@
+              <Divider style={styles.divider} />
+              <Text style={styles.descriptionText}>
+                Ingredients: {props.Dish.ingredients}
+              </Text>
+              <Divider style={styles.divider} />
+              <Text style={styles.descriptionText}>
+                Estimated Time: {props.Dish.timeString}
+              </Text>
+              <View style={styles.spacer}></View>
+              <Reviews
+                rating={props.Dish.rating}
+                numReviews={props.Dish.numReviews}
+                reviews={first5Reviews}
+                chefid={props.Dish.chefid}
+                dishid={props.Dish.dishid}
+              />
+              <View style={styles.spacer} />
+              <View style={styles.spacer} />
             </View>
-            
-        </Modal>
-    );
+          </View>
+        </ScrollView>
+        <View style={styles.closeButton}>
+          <Button
+            onPress={() => close(setVisible, props.hideModal)}
+            buttonStyle={styles.closeButtonStyle}
+            icon={
+              <Icon
+                name="close"
+                size={25}
+                color="white"
+                style={{
+                  backgroundColor: "black",
+                  borderRadius: 15,
+                  outline: "white solid 1px"
+                }}
+              />
+            }
+          />
+        </View>
+        <View style={styles.checkout}>
+          <Button
+            type="solid"
+            title=" - "
+            onPress={() => remove(count, setCount)}
+            titleStyle={styles.buttonText}
+            buttonStyle={styles.minusButton}
+          />
+          <Text style={styles.countStyle}>{count}</Text>
+          <Button
+            type="solid"
+            style={styles.plusButtonPadding}
+            title=" + "
+            onPress={() => setCount(count + 1)}
+            titleStyle={styles.buttonText}
+            buttonStyle={styles.plusButton}
+          />
+          <Button
+            type="solid"
+            title="Add to Cart"
+            titleStyle={styles.addToCartText}
+            buttonStyle={styles.addToCartButton}
+            icon={
+              <Icon
+                style={styles.cartPadding}
+                name="basket"
+                type="simple-line-icon"
+                size={20}
+                color="white"
+              />
+            }
+            onPress={() => {
+              if (count > 0) {
+                const found = global.cart.findIndex(
+                  item => item["dish"] == props.Dish
+                );
+                if (found < 0) {
+                  const dish = props.Dish;
+                  addToCart({ dish, count });
+                } else {
+                  addQuantity(found, count);
+                }
+                Alert.alert(
+                  "Added to Cart",
+                  "Successfully added items to cart!"
+                );
```

Comment:
Added the above onPress functionality to add a dish to the cart and indicate with an alert